src/v/config/configuration.cc

```diff
@@ -1653,7 +1653,16 @@ configuration::configuration()
         model::cloud_credentials_source::sts,
         model::cloud_credentials_source::gcp_instance_metadata,
         model::cloud_credentials_source::azure_aks_oidc_federation,
+        model::cloud_credentials_source::azure_vm_instance_metadata,
       })
+  , cloud_storage_azure_managed_identity_id(
+      *this,
+      "cloud_storage_azure_managed_identity_id",
+      "user managed identity id to be used with "
+      "cloud_storage_credential_source: azure_vm_instance_metadata",
```

Comment:
>used with cloud_storage_credential_source: azure_vm_instance_metadata

Please write out a more verbose and formal description. Something like:

"The managed identity ID to use with Azure Managed Identities. This takes affect when the cloud_storage_credential_source configuration option is set to azure_vm_instance_metadata."

This really helps the docs team.

Reply:
thanks, fixed